walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -6,15 +6,15 @@
 
 = Running the mobile showcase application with {mobile-services-name}
 
-This Solution Pattern demonstrates running the mobile showcase application using the services provided by Mobile Developer Services.
+This Solution Pattern demonstrates running the GraphQL PWA Showcase Application using Red Hat Mobile Developer Services.
 
-It will show you how to:
+Solution Pattern will show you how to:
 
-* Build and run the link:{showcase-app-link}[Ionic mobile showcase app].
-* Deploy the data sync server of the mobile showcase app.
-* Protect the showcase mobile app and the backend service using the Identity Management service.
+* Build and run the link:{showcase-app-link}[Ionic Mobile Showcase app].
```

Comment:
may be we just call it `GraphQL client app`?

Reply:
It has push integration so showcase still stands :)